src/cluster/cluster.cc

```diff
@@ -25,10 +25,10 @@
 #include <cstring>
 #include <memory>
 
+#include "commands/redis_cmd.h"
```

Comment:
Does C++ module style prefer UpperCase? That is, "Command/redis_cmd.h".

Reply:
Besides, prefer "command" to "commands".

Reply:
I think we need to unify the style of filenames, if it is `Command`, maybe it will be `Command/RedisCmd.h`, so I think we can keep underscore-style.

Reply:
> I think we need to unify the style of filenames, if it is Command, maybe it will be Command/RedisCmd.h, so I think we can keep underscore-style.

Make sense :)

Reply:
> Does C++ module style prefer UpperCase? That is, "Command/redis_cmd.h".

ClickHouse src dir uses uppercase but some of dir is the snake case, I can't say which one is better, but I prefer the snake case than the uppercase from my own habits.